### PR TITLE
Calling participant list pane for mobile

### PR DESCRIPTION
**Pull request by mgamis-msft**
# What

Full screen pane showing participants when ParticipantsButton is clicked in Call Composite in mobile form factor.

Here is a video of local sample testing:
https://user-images.githubusercontent.com/79475487/153607270-b4025472-a84c-44cb-a2b9-265c4cfd577c.mp4

This PR is a little rough but I just wanted to get some quick feedback

# Why

https://skype.visualstudio.com/SPOOL/_workitems/edit/2757257
https://skype.visualstudio.com/SPOOL/_workitems/edit/2590657

# How Tested

On local calling sample. Should also generate new UI snapshot for participant button test for mobile.

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-composites/src/composites/CallComposite/components/CallControls.tsx [hunk 1]
```diff
@@ -163,6 +172,13 @@ export const CallControls = (props: CallControlsProps): JSX.Element => {
   );
 };
 
+const isDisabled = (option?: boolean | { disabled: boolean }): boolean => {
+  if (option === undefined || option === true || option === false) {
```
Comment (alkwa-msft) [inline packages/react-composites/src/composites/CallComposite/components/CallControls.tsx:176]: can we do something like

c = a?.b
return c === undefined ? false : c

File: packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx [hunk 2]
```diff
@@ -43,33 +44,46 @@ export const CallArrangement = (props: CallArrangementProps): JSX.Element => {
...
+        <Stack.Item styles={callGalleryStyles} grow>
+          {props.onRenderGalleryContent && (
+            <Stack verticalFill styles={mediaGalleryContainerStyles}>
+              {props.onRenderGalleryContent()}
+            </Stack>
+          )}
         </Stack.Item>
-      )}
+
+        {props.callControlProps?.options !== false && (
```
Comment (alkwa-msft) [inline packages/react-composites/src/composites/CallComposite/components/CallArrangement.tsx:78]: [feel free to disregard]
might be good to define a new variable to say why we are showing the callControls or not.

File: packages/communication-react/review/communication-react.api.md [hunk 3]
```diff
@@ -1120,7 +1122,11 @@ export const DEFAULT_COMPOSITE_ICONS: {
     OptionsCamera: JSX.Element;
     OptionsMic: JSX.Element;
     OptionsSpeaker: JSX.Element;
-    ParticipantItemMicOff: JSX.Element;
+    ParticipantItemMicOff: JSX.Element; /**
+    * Icons that can be overridden for {@link CallComposite}.
```
Comment (JamesBurnside) [inline packages/communication-react/review/communication-react.api.md:1126]: @prprabhu-ms it's happening again!! 🤯

This comment is a) out of place and b) should never exist in this file.
This happened before in one of my local builds -- I manually removed it (it doesn't happen in the fresh CI build) - I couldn't get to the bottom of _why_ it happened

File: packages/react-composites/src/composites/CallComposite/Strings.tsx [hunk 4]
```diff
@@ -173,4 +173,12 @@ export interface CallCompositeStrings {
    * Tooltip text used to inform a user that toggling microphone in lobby is not supported.
    */
   microphoneToggleInLobbyNotAllowed: string;
+  /**
+   * Title used in pane showing all participants in call
+   */
+  peoplePaneTitle: string;
```
Comment (JamesBurnside) [inline packages/react-composites/src/composites/CallComposite/Strings.tsx:179]: @prprabhu-ms @PorterNan is this the kind of feature that should go behind beta or not?

**Comment by JamesBurnside** [comment]
Looks good - just holding off approval until e2e builds complete to review any visual changes

**Comment by prprabhu-ms** [comment]
I'm confused by this PR. Let's wait for discussion at https://teams.microsoft.com/l/message/19:5e150d250f724d1089c949305efdc8cc@thread.skype/1644608224641?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e9c1fc3-39df-4486-a26a-456d80e80f82&parentMessageId=1644608224641&teamName=Azure%20Communication%20Services&channelName=Web%20UI%20-%20Team&createdTime=1644608224641 to resolve before landing this.

File: packages/communication-react/review/communication-react.api.md (hunk 3 above)
Comment (mgamis-msft) [reply packages/communication-react/review/communication-react.api.md]: Thanks for the call out. I manually removed it. Not sure why this is happening either.

**Comment by mgamis-msft** [comment]
> I'm confused by this PR. Let's wait for discussion at https://teams.microsoft.com/l/message/19:5e150d250f724d1089c949305efdc8cc@thread.skype/1644608224641?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e9c1fc3-39df-4486-a26a-456d80e80f82&parentMessageId=1644608224641&teamName=Azure%20Communication%20Services&channelName=Web%20UI%20-%20Team&createdTime=1644608224641 to resolve before landing this.

Holding off this PR after all major UX changes are added to Meeting Composite

**Pull request closed without merging**